src/components/TheHeader.vue

```diff
@@ -72,6 +119,15 @@ export default {
     isLoggedIn() {
       return this.$store.getters.isLoggedIn;
     },
+    line1active() {
+      return { "line1--active": this.mobileMenuOn };
+    },
+    line2active() {
+      return { "line2--active": this.mobileMenuOn };
+    },
+    line3active() {
+      return { "line3--active": this.mobileMenuOn };
+    },
```

Comment:
This looks quite repetitive - what do you think about adding a class to the container wrapping the lines instead and then handling all of the lines in your styles in case the container is active? 🤔
Only one class needed then 😄

Reply:
point taken 💛  single class now handling lines